go/vt/vtgate/planbuilder/operator_transformers.go

```diff
@@ -59,18 +59,36 @@ func transformToLogicalPlan(ctx *plancontext.PlanningContext, op ops.Operator, i
 		return transformCorrelatedSubQueryPlan(ctx, op)
 	case *operators.Derived:
 		return transformDerivedPlan(ctx, op)
+	case *operators.SimpleProjection:
+		src, err := transformToLogicalPlan(ctx, op.Source, false)
+		if err != nil {
+			return nil, err
+		}
+
+		return &simpleProjection{
+			logicalPlanCommon: newBuilderCommon(src),
+			eSimpleProj: &engine.SimpleProjection{
+				Cols: op.Columns,
+			},
+		}, nil
```

Comment:
Let's extract this to its own function so we can have a one line `return` in each `case`.